docs/run-core-node/tier-1-orgs.mdx

```diff
@@ -3,39 +3,45 @@ title: Tier 1 Organizations
 sidebar_position: 120
 ---
 
-To help with Stellar’s decentralization, the most reliable and advanced Stellar teams join the ranks of “Tier 1 Organizations.” These organizations run three validators, coordinate any changes to their quorumsets, and hold themselves to a higher standard of uptime and responsiveness.
+To help with Stellar’s decentralization, the most advanced teams building on Stellar run validators and strive to join the ranks of “Tier 1 organizations.”
 
-SDF works closely with Tier 1 Orgs to ensure the health of the network, maintain good quorum intersection, and build in redundancy to minimize network disruptions. This guide outlines what it takes to be a Tier 1 Org.
+Tier 1 organizations are the organizations that are members of minimal quorums in the core of the Stellar network. This is an emergent property of the Stellar network that depends on the collective QSet configurations of the validators in the Stellar network.
```

Comment:
Can we link out to a definition or glossary term for "QSet"? Or otherwise define it the first time it's used.

Reply:
I was trying to not use "quorum set" because I think that term is very confusing and we should get rid of it. However, given how pervasive it is in our docs and code, that might be a lost cause. Should I just go with "quorum set"?

Reply:
I'll just go with "quorum set" for now and we can discuss this later.